README.md

```diff
@@ -43,9 +43,47 @@ steps:
     with:
       # The report file must be there, otherwise Code Climate won't find it
       coverageCommand: mvn test
+      coverageLocations: ${{github.workspace}}/target/site/jacoco/jacoco.xml:jacoco
+```
+
+#### Example of multiple test coverages for monorepo with Jest
+
+Let's say you have a monorepo with two folders `client` and `server` both with separated coverage folders and `yarn coverage` script which runs jest within both folders.
+
+```json
+"scripts": {
+  "coverage": "yarn client coverage && yarn server coverage"
+}
+```
+
+First be sure that paths in your `coverage/lcov.info` are correct. Means they should be either absolute or relative *to the root of the monorepo*. Open `lcov.info` and search for any path. For example:
+
+```lcov
+SF:src/server.ts
+```
+
+If you find a *relative* path like this (happens for Jest 25+) it's not correct. The path must be relative to the monorepo root. Means it should be `server/src/server.ts`. For Jest you can set the root of your repo like this:
+
+```javascript
+// server/jest.config.js
+module.exports = {
+  ...
+  coverageReporters: [['lcov', { projectRoot: '..' }]]
+  ...
+};
+```
+
+```yaml
+steps:
+  - name: Test & publish code coverage
+    uses: paambaati/codeclimate-action@v2.5.7
+    env:
+      CC_TEST_REPORTER_ID: ${{secrets.CC_TEST_REPORTER_ID}}
+    with:
+      # coverageCommand: yarn coverage - this line is actually not needed because `yarn coverage` is by default
```

Comment:
@MartinNuc Is this documented?

Reply:
Yes the default value is documented in the table here https://github.com/paambaati/codeclimate-action#inputs